scripts/globals/effects/inner_strength.lua

```diff
@@ -0,0 +1,17 @@
+-----------------------------------
+--
+--     tpz.effect.INNER_STRENGTH
+--
+-----------------------------------
+function onEffectGain(target, effect)
+    target:addMod(tpz.mod.HPP, 100)
+    target:addMod(tpz.mod.PERFECT_COUNTER_ATT, 100)
+end
+
+function onEffectTick(target, effect)
+end
+
+function onEffectLose(target, effect)
+    target:delMod(tpz.mod.HPP, 100)
+    target:addMod(tpz.mod.PERFECT_COUNTER_ATT, -100)
```

Comment:
should be delmod matching the addmod above, not a 2nd addmod with a negative amount

Reply:
changed to delmod 100 not addmod -100